{"uuid":"3db92057-21c9-43ce-83b6-591f2890370c","historyId":"42bd0695bad4e019ced0718bc51fa2ce","testCaseId":"[engine:junit-jupiter]/[class:org.academo.academo.London.service.TaskServiceTest]/[method:getAllTasks_DatabaseError()]","testCaseName":"Get all tasks - Database Error (DatabaseServiceException)","fullName":"org.academo.academo.London.service.TaskServiceTest.getAllTasks_DatabaseError","labels":[{"name":"tag","value":"unit"},{"name":"junit.platform.uniqueid","value":"[engine:junit-jupiter]/[class:org.academo.academo.London.service.TaskServiceTest]/[method:getAllTasks_DatabaseError()]"},{"name":"host","value":"Rockio"},{"name":"thread","value":"68025@Rockio.main(3)"},{"name":"framework","value":"junit-platform"},{"name":"language","value":"java"},{"name":"package","value":"org.academo.academo.London.service.TaskServiceTest"},{"name":"testClass","value":"org.academo.academo.London.service.TaskServiceTest"},{"name":"testMethod","value":"getAllTasks_DatabaseError"},{"name":"suite","value":"Task service test"}],"links":[],"name":"Get all tasks - Database Error (DatabaseServiceException)","status":"passed","stage":"finished","description":"","steps":[],"attachments":[],"parameters":[],"start":1757418976922,"stop":1757418976924}